lib/datasets/wikipedia-kyoto-japanese-english.rb

```diff
@@ -114,6 +119,14 @@ def download_tar_gz
       data_path
     end
 
+    def with_increased_entity_expansion_text_limit
+      default_limit = REXML::Security.entity_expansion_text_limit
+      REXML::Security.entity_expansion_text_limit = ENTITY_EXPANSION_TEXT_LIMIT
+      yield
+    ensure
+      REXML::Security.entity_expansion_text_limit = default_limit
+    end
```

Comment:
Hmm. We can use this for now but it seems that `REXML::Parsers::StreamParser#entity_expansion_text_limit=` should exist for local limitation change. Could you propose it to ruby/rexml?

Reply:
It sounds pretty nice. Sure!

Reply:
I've proposed it at https://github.com/ruby/rexml/issues/192.